Utils/API/server/lib/dkb/api/storages/es.py

```diff
@@ -28,6 +28,9 @@
 # Default datetime format
 DATE_FORMAT = '%d-%m-%Y %H:%M:%S'
 
+with open('ctag_formats.json') as f:
+    OUTPUT_FORMATS = json.load(f)
```

Comment:
Done.

Reply:
Note: usage of additional query to ES instead of json file representing Oracle leads to a single difference in results for values used for testing (project `data18_13TeV`, list of tags consisting of one tag `p3553`): ES contains tasks with `p3553` tag and output datasets with `DAOD` format, while Oracle's data yields no such output corresponding to this tag. I doubt that this is a problem, since Oracle produces this format when presented with some other tags (`p3166`, for example), hence it does not looks to me as some special format that should be avoided.

Reply:
It must be due to the way `"data_format"` field is filled: it contains list of both "real" format ("DAOD_SMTH") and "main" format ("DAOD") (see [here](https://github.com/PanDAWMS/dkb/blob/c5326669ce489478a35d8c096152446ba37281a2/Utils/Dataflow/093_datasetsFormat/datasets_format.py#L48) for details).

Not sure how to handle it for now, will take a closer look ASAP.